src/gui/advancedsettings.h

```diff
@@ -58,7 +58,7 @@ private slots:
     template <typename T> void addRow(int row, const QString &rowText, T* widget);
 
     QLabel labelQbtLink, labelLibtorrentLink;
-    QSpinBox spin_cache, spin_save_resume_data_interval, outgoing_ports_min, outgoing_ports_max, spin_list_refresh, spin_maxhalfopen, spin_tracker_port, spin_cache_ttl,
+    QSpinBox spin_aio_threads, spin_cache, spin_save_resume_data_interval, outgoing_ports_min, outgoing_ports_max, spin_list_refresh, spin_maxhalfopen, spin_tracker_port, spin_cache_ttl,
```

Comment:
camelCase!
Don't look at the legacy code, we will fix it with time. Just read the Coding Guidelines.

Reply:
Sorry, I will fix it up.